custom/expired_domains.md

```diff
@@ -0,0 +1,4 @@
+This document contains a list of domains for which we have removed custom dev work and the git commits
+in which the custom code was deleted.
+
+* care-ihapc-live: 215c9aa716d5572b2dc97bfc3587d0f80111db86
```

Comment:
@dimagi/team-commcare-hq FYI keeping a list of git hashes here for custom code that gets deleted in case we need to restore it again.